paymentsheet/src/main/java/com/stripe/android/link/ui/FullScreenContent.kt

```diff
@@ -67,39 +71,49 @@ internal fun FullScreenContent(
         navigationChannel = navigationChannel,
         navHostController = navController,
         keyboardController = keyboardController,
-        onBackStackEntryUpdated = onNavBackStackEntryChanged
+        onBackStackEntryUpdated = onNavBackStackEntryChanged,
+        onPopBackStackResult = { handled ->
+            if (!handled) {
+                dismiss()
+            }
+        }
     )
 
-    Box(
-        modifier = modifier
+    ElementsBottomSheetLayout(
+        state = bottomSheetState,
+        onDismissed = dismiss,
```

Comment:
This won't be called on phones, since the sheet is currently filling the entire height, but it might be called on larger screens when the user taps outside the sheet. Might be worth preventing that while API calls are in-progress…

Reply:
Addressed in 3aa872a1c95f3ed7ad6648ac0c6ffdea746c1218.

Reply:
And soon useful if we make the sheet dynamic in height! 👏